src/components/search-query.jsx

```diff
@@ -1,83 +1,95 @@
 import React from 'react';
```

Comment:
Now that this component is nested in the SearchQuery folder, we can delete this file from the main `components` folder.